src/controllers/confirmation.controller.ts

```diff
@@ -16,6 +16,7 @@ export const get = (req: Request, res: Response) => {
   deleteApplicationData(req.session);
 
   return res.render(CONFIRMATION_PAGE, {
+    ...appData,
```

Comment:
I would have just created another variable for `entity.email` instead of passing the entire object.